docs/_articles/writing-style.md

```diff
@@ -0,0 +1,15 @@
+---
+title: UI text writing style
+---
+
+The rules are:
+
+- Capitalize first letter of everything - from single words to sentences.
+- Finish with a period only if it's a full sentence - e.g. not if it's a heading, short instruction, etc.
+- Always make punctuation part of the translated string (so that RTL languages don't start with a period, for example).
+- Use real ellipsis character (<code>…</code>) rather than three periods (<code>...</code>)<sup>1</sup>.
+- Remember that our name is capitalized like this: `KoboToolbox`.
+
+If in doubt, please mimick what is already there.
```

Comment:
No "k" in the standard spelling of "mimic"

Reply:
Fixed :ok: